test/OpenTelemetry.Tests/Metrics/MetricApiTestsBase.cs

```diff
@@ -51,6 +65,10 @@ public void MeasurementWithNullValuedTag()
         using var meter = new Meter(Utils.GetCurrentMethodName());
         var exportedItems = new List<Metric>();
         using var meterProvider = Sdk.CreateMeterProviderBuilder()
+            .ConfigureServices(services =>
+            {
+                services.AddSingleton(this.configuration);
+            })
             .AddMeter(meter.Name)
             .AddInMemoryExporter(exportedItems)
```

Comment:
No biggie if you don't want to do take this on, but seems like these lines are repeated a bajillion times in this file. Might be a good extract method opportunity.

Reply:
> Bajillion is an informal word that means a very large but unspecified number. It is not a real number, but a way of exaggerating or expressing a huge quantity. For example, one might say “I have a bajillion things to do today” to mean that they have a lot of tasks. [The word bajillion is probably a blend of bazillion and jillion, which are also informal words for a large number](https://www.merriam-webster.com/dictionary/bajillion)[1](https://www.merriam-webster.com/dictionary/bajillion)[2](https://en.wikipedia.org/wiki/Indefinite_and_fictitious_numbers). There is no standard way to write bajillion in numerals, since it is not a precise value. However, some people might use a 1 followed by many zeros, such as 1,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000, or use scientific notation, such as 10100
>  or 10googol
> , where googol is another name for 10100
> . However, these are just arbitrary representations, and there is no official or correct way to write bajillion.

Learned something new today!

Reply:
@utpilla Don't worry about refactoring it on here. On #4958 I'm refactoring how these tests create their providers. Example: https://github.com/open-telemetry/opentelemetry-dotnet/pull/4958/files#diff-ae011cf18b74623dd9376d03c8629da33ab122c4d4b6a6c9d3f54325a13637eeR44

I'll have to merge these changes over there and I can encapsulate the config into the helper to achieve what @alanwest  is asking for.

Reply:
> 1,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000

Ok, I'll admit I exaggerated a bit... This prolly woulda broken GitHub.